autoload/clap/installer.vim

```diff
@@ -30,8 +32,6 @@ function! s:run_term(cmd, cwd, success_info) abort
           \})
   endif
 
-  let bufnr = bufnr('')
-
   normal! G
```

Comment:
This line should be placed before bufnr is used?

Reply:
Not sure, the current one works for me, both vim and neovim. If this change is helpful for you, I'm ok to include.